mne_bids_pipeline/steps/preprocessing/__init__.py

```diff
@@ -4,22 +4,24 @@
 from . import _02_head_pos
 from . import _03_maxfilter
 from . import _04_frequency_filter
-from . import _05_make_epochs
+from . import _05_regress_artifact
 from . import _06a_run_ica
 from . import _06b_run_ssp
-from . import _07a_apply_ica
-from . import _07b_apply_ssp
-from . import _08_ptp_reject
+from . import _07_make_epochs
+from . import _08a_apply_ica
+from . import _08b_apply_ssp
+from . import _09_ptp_reject
 
 _STEPS = (
     _01_data_quality,
     _02_head_pos,
     _03_maxfilter,
     _04_frequency_filter,
-    _05_make_epochs,
+    _05_regress_artifact,
```

Comment:
Previously the order was SSS, filt, epoch, SSP/ICA, clean epochs. Now the order is SSS, filt, SSP/ICA, epoch, clean epochs. This makes it clearer that the epochs *are not* used for SSP and ICA, which is helpful in terms of understanding the flow in that we basically stay working with raw data ountil step _07 now.